nixos/lib/test-driver/Machine.pm

```diff
@@ -43,7 +43,7 @@ sub new {
             if defined $args->{bios};
         $startCommand .= $args->{qemuFlags} || "";
     } else {
-        $startCommand = Cwd::abs_path $startCommand;
+        #$startCommand = Cwd::abs_path $startCommand;
```

Comment:
Not sure what the goal of the `abs_path` was here, but it doesn't seem to be designed to handle composite commands (of the sort the other `if` branch generates) so I killed it for now. I obviously wouldn't merge until we figure out if the current behavior is needed.